src/main/kotlin/it/pagopa/wallet/services/RedisResumePolicyService.kt

```diff
@@ -0,0 +1,42 @@
+package it.pagopa.wallet.services
+
+import it.pagopa.wallet.config.properties.RedisResumePolicyConfig
+import java.time.Duration
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class RedisResumePolicyService(
```

Comment:
```suggestion
class ResumePolicyService(
```

this service could be abstracted from redis, we could decide to save it on another storage and in this way we could leave the interfaces unchanged

Reply:
I agree with you, RedisResumePolicyService is already the implementation of the interface ResumePolicyService, that's why has been named with that specific naming